views/index.html.twig

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="robots" content="noindex,nofollow" />
+    <link rel="shortcut icon" type="image/png" href="/satis/views/assets/img/favicon.png">
```

Comment:
This absolute path is specific to your deployment. The file must be duplicated into the "output" directory and the path must be relative.

Reply:
Yeah @GromNaN is correct. Satis is basically a static site generator. The output directory is configurable, but any assets should be placed there along with the generated html. Alternatively, you could inline the png like we did with most other javascript/css/image assets. See https://stackoverflow.com/a/5568484 for an example.